src/web/topic/components/TopicWorkspace/MoreActionsDrawer.tsx

```diff
@@ -342,35 +343,37 @@ export const MoreActionsDrawer = ({
               </ToggleButton>
             </ListItem>
 
-            <ListItem disablePadding={false}>
+            <ListItem disablePadding>
               <Typography variant="body2">Layout Thoroughness</Typography>
               <Tooltip
-                title="Determines how much effort the layout algorithm puts into laying out nodes such that they efficiently use space. 1 = lowest effort, 100 = highest effort."
-                enterTouchDelay={0} // allow touch to immediately trigger
-                leaveTouchDelay={Infinity} // touch-away to close on mobile, since message is long
+                title="Determines how much effort the layout algorithm puts into laying out nodes such that they efficiently use space. Low = minimal effort, High = maximum effort."
+                enterTouchDelay={0} // Trigger immediately on touch
+                leaveTouchDelay={Infinity} // Close on touch-away
```

Comment:
We should keep "since message is long" in the comment here - it explains why we're using touch-away to close, rather than the default timeout for Mui Tooltips